src/GeneralStateTestsFiller/stExtCodeHash/extCodeHashCreatedAndDeletedAccountStaticCallFiller.json

```diff
@@ -0,0 +1,68 @@
+{
+  "extCodeHashCreatedAndDeletedAccountStaticCall": {
+    "_info": {
+      "comment": "EXTCODEHASH/EXTCODESIZE of an account created then deleted via STATICCALL in same transaction"
+    },
+    "env": {
+      "currentCoinbase": "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+      "currentDifficulty": "0x20000",
+      "currentGasLimit": "1000000",
+      "currentNumber": "1",
+      "currentTimestamp": "1000",
+      "previousHash": "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+    },
+    "expect": [
+      {
+        "indexes": {
+          "data": -1,
+          "gas": -1,
+          "value": -1
+        },
+        "network": [
+          ">=Constantinople"
+        ],
+        "result": {
+          "deadbeef00000000000000000000000000000000": {
```

Comment:
so if in previous test we were checking that address "shouldnotexist": "1"
here in this case we need to check that it does exist. cause staticcall would fail to perform selfdestruct

Reply:
Added.